.github/workflows/ci.yml

```diff
@@ -11,7 +11,7 @@ jobs:
       - uses: actions/checkout@v1
       - uses: actions-rs/toolchain@v1
         with:
-          toolchain: 1.44.0
+          toolchain: stable
```

Comment:
This conflicts with #62, where the MSRV is set to 1.47.0.

Reply:
Resolved to 1.47.0